tools/mcstas_to_nexus.ipynb

```diff
@@ -221,7 +221,9 @@
     "            values=event_index.coords[\"event_time_zero\"]\n",
     "            .to(unit=monitor_data[\"event_time_zero\"].attrs[\"units\"], copy=False)\n",
     "            .values.astype(int),\n",
-    "        )"
+    "        )\n",
+    "    # Close the file after writing\n",
+    "    f.close()"
```

Comment:
Can't you use a context manager instead? It looks like this is used in only one cell.